### ISSUE TITLE
Export configuration

### ISSUE THREAD

**Issue by uGeek** [vfsfitvnm/ViMusic#8]
### Feature description

Would it be possible to create a file to export the configuration and be able to import it on another device?

This file would contain the information of:
-Your Playlist
-Favourites
-History
-Most played

### Why do you want this feature?

This would be fantastic because we could use vimusic keeping, lists, favorites,... on the same device if you restore the phone or on multiple devices

### Additional information

Congratulations for this great application.

I have been using it since the first version and it is by far the best, both in terms of interface, speed and usability.

Thank you very much

**Comment by vfsfitvnm** [comment]
Working on it :mechanical_arm: 

<img src="https://user-images.githubusercontent.com/46219656/172443420-4a0b5562-3a97-4c75-8e46-9ef30cd82aa3.jpg" width="200" />

**Comment by uGeek** [comment]
Great!!! It has a spectacular look

Thank you very much and looking forward to trying it

**Comment by vfsfitvnm** [comment]
`v0.1.3` is now live. Please let me know if there are problems

**Comment by uGeek** [comment]
It works excellently. I have exported from my mobile and imported on another mobile and tablet and it works perfectly.

Thank you very much, it's amazing

I was thinking of another suggestion attached to this one.

I have saved the file to a cloud that I have mounted with the WebDAV app from https://www.davx5.com/. It allows you to mount your cloud just like Google Drive, Nextcloud,...

I have noticed that ViMusic has permissions to save and read files on Android mounted systems, I have restored from cloud mounted ViMusic on my other devices.

**Suggestion**
It could have a third synchronization option, where ViMusic exports exactly the same database, but this time instead of the name including the date, it is a fixed name, that whenever we hit the synchronization button, ViMusic will see if the file is newer than the current version and update "synchronize".

In this way, all the lists, favorites,... through an external application, would allow ViMusic to be synchronized.

This right now is not an important option, since through the export-import option it can be done manually. But it is an interesting and perhaps simple feature to have all your music in sync.

Thank you very much and the application is simply amazing
